### ISSUE TITLE
Force the cluster to recycle all nodes regularly

### ISSUE THREAD

**Issue by digitalronin** [ministryofjustice/cloud-platform#1153]
If pods stick around forever, then things like namespace quota and limit range changes may not have the desired effects, because the cluster is full of pods which are still holding reservations for the old values.

Create a mechanism whereby all nodes are gracefully replaced, every so often, by replacing the oldest worker node, periodically.

 - [ ] Comms: Tell users before we start doing this for real
 - [ ] Where does this run?

**Comment by vijay-veeranki-moj** [comment]
https://github.com/ministryofjustice/cloud-platform-infrastructure/pull/442.

Created a script to recycle-old worker node.

**Comment by vijay-veeranki-moj** [comment]
Created new ticket to run the script created periodically.

https://app.zenhub.com/workspaces/cloud-platform-team-5ccb0b8a81f66118c983c189/issues/ministryofjustice/cloud-platform/1336
